src/content/guidelines/motion/choreography.mdx

```diff
@@ -65,18 +64,18 @@ Elements of our interface dance on the Grid. Motion paths trace lines along the
 
 ## Composition
 
-When multiple animated elements coexist or interact with each other within the same view, it is vital to make the many moving elements work together and form coherent experience, to better provide way-finding and focus to guide the user experience.
+When multiple animated elements coexist or interact with each other within the same view, it is vital to make the many moving elements work together and form a coherent experience, to better provide way-finding and focus.
 
 ### Consistency
 
 #### Semantic consistency
 
-When elements convey the same meaning, or perform the same functions, use the same motion for them. Similarly, actions with dramatically different meaning and intent should carry different motions. This helps to reinforce the meaning behind a motion, and improves user’s proficiency with the interface.
+When elements convey the same meaning or perform the same function, use the same motion for them, and vice-versa. This helps to reinforce the meaning behind a movement and improves the user’s understanding of the interface.
```

Comment:
How about "proficiency using the interface"? The point here is with a clearer/quicker understanding of what everything means, people can work faster, like a pro.